packages/fxa-auth-server/lib/tokens/key_fetch_token.js

```diff
@@ -21,21 +20,17 @@ module.exports = function(log, Token) {
 
   KeyFetchToken.tokenTypeID = 'keyFetchToken';
 
-  KeyFetchToken.create = function(details) {
+  KeyFetchToken.create = async function(details) {
     log.trace('KeyFetchToken.create', { uid: details && details.uid });
-    return Token.createNewToken(KeyFetchToken, details || {}).then(token => {
-      return token.bundleKeys(details.kA, details.wrapKb).then(keyBundle => {
-        token.keyBundle = keyBundle;
-        return token;
-      });
-    });
+    const token = await Token.createNewToken(KeyFetchToken, details || {});
+    const keyBundle = await token.bundleKeys(details.kA, details.wrapKb);
+    token.keyBundle = keyBundle;
+    return token;
   };
 
-  KeyFetchToken.fromId = function(id, details) {
+  KeyFetchToken.fromId = async function(id, details) {
     log.trace('KeyFetchToken.fromId');
-    return P.resolve(
-      new KeyFetchToken({ id, authKey: details.authKey }, details)
-    );
+    return new KeyFetchToken({ id, authKey: details.authKey }, details);
```

Comment:
Unnecessary `async` since I don't see an `await`.

Reply:
This function previously returned a Promise, so I had to make the function `async` so that this would match the API. Am I missing something?

Reply:
No, you are not missing anything.  There are probably still `.then`s chaining off of it.

r+